exercises/robot-name/robot-name.jl

```diff
@@ -0,0 +1,11 @@
+type Robot
```

Comment:
I decided to leave empty structure and method skeletons. Created guidelines with small help. :)